README.md

```diff
@@ -0,0 +1,88 @@
+[![CocoaPods](https://img.shields.io/cocoapods/v/Iterable-iOS-SDK.svg?style=flat)](https://cocoapods.org/pods/Iterable-iOS-SDK)
+[![License](https://img.shields.io/cocoapods/l/Iterable-iOS-SDK.svg?style=flat)](https://opensource.org/licenses/MIT)
+[![Docs](https://img.shields.io/cocoapods/metrics/doc-percent/Iterable-iOS-SDK.svg?style=flat)](http://cocoadocs.org/docsets/Iterable-iOS-SDK)
+[![Build Status](https://travis-ci.com/Iterable/swift-sdk.svg?branch=master)](https://travis-ci.com/Iterable/swift-sdk)
+
+# Iterable iOS SDK
+
+`Iterable-iOS-SDK` is a Swift implementation of an iOS client for Iterable, for iOS versions 9.0 and higher.
+
+# Setting up a push integration in Iterable
+
+Before you even start with the SDK, you will need to 
+
+1. Set your application up to receive push notifications, and 
+2. Set up a push integration in Iterable. This allows Iterable to communicate on your behalf with Apple's Push Notification Service
+
+If you haven't yet done so, you will need to enable push notifications for your application. This can be done by toggling `Push Notifications` under your target's `Capabilities` in Xcode. You can also do it directly in the app center on Apple's member center; go to `Identifiers -> App IDs -> select your app`. You should see `Push Notifications` under `Application Services`. Hit `Edit` and enable `Push Notifications`.
+
+You will also need to generate an SSL certificate and private key for use with the push service. See the links at the end of this section for more information on how to do that.
+
+Once you have your APNS certificates set up, go to `Integrations -> Mobile Push` in Iterable. When creating an integration, you will need to pick a name and a platform. The name is entirely up to you; it will be the `appName` when you use `registerToken` in our SDK. The platform can be `APNS` or `APNS_SANDBOX`; these correspond to the production and sandbox platforms. Your application will generate a different token depending on whether it is built using a development certificate or a distribution provisioning profile.
+
+![Creating an integration in Iterable](http://support.iterable.com/hc/en-us/article_attachments/202957719/Screen_Shot_2015-07-30_at_3.15.56_PM.png)
```

Comment:
Are you going to change the documentation in this PR or a new one to remove the Iterable specific parts.

Reply:
I want to create a separate one for docs. For this one, I just want to fix the podspec etc.